### PR TITLE
add initial content for release consolidation

### PR DESCRIPTION
**Pull request by Siegfriedk**
This PR combines the content of the website and tractus-x-release repository into sig-release.

It thereby fixes the [Issue #572](https://github.com/eclipse-tractusx/sig-release/issues/572)

See:
- https://github.com/eclipse-tractusx/tractus-x-release/pull/26
- https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/852

Superseeds https://github.com/eclipse-tractusx/sig-release/pull/519

### CODE REVIEW

File: README.md
Comment (RolaH1t) [inline README.md:13]: Tractus-X operates on a quarterly release cycle. Each quarter, a new [tractus-x release] (tractus-x-release.md) following [CalVer](https://calver.org) is released.

File: docs/product_release.md
Comment (RolaH1t) [inline docs/product_release.md:1]: in general: pay attention to CAPITAL or small letters => especially when referencing URLs and documents

File: docs/product_release.md
Comment (RolaH1t) [inline docs/product_release.md:52]: consider to add semver URL

File: docs/tractus-x-release.md
Comment (RolaH1t) [inline docs/tractus-x-release.md:3]: Tractus-X operates on a quarterly release cycle. Each quarter, a new tractus-x release following [CalVer] (URL) is released. Each tractus-x release contains multiple [products] (product_release.md).

File: docs/tractus-x-release.md
Comment (RolaH1t) [inline docs/tractus-x-release.md:38]: this helpful graph shall soon be published within a document of the CX Association => we can add the URL as soon as available

File: docs/tractus-x-release.md
Comment (RolaH1t) [inline docs/tractus-x-release.md:40]: Example for the Catena-X Operation System ("cxOS")

File: docs/tractus-x-release.md
Comment (RolaH1t) [inline docs/tractus-x-release.md:42]: No matter if carry-over product versions of a previous TX release or product evolutions, developed within the current iteration.

File: docs/tractus-x-release.md
Comment (RolaH1t) [inline docs/tractus-x-release.md:44]: Any findings are looped back to the [product development teams] (product_release.md)

File: docs/tractus-x-release.md
Comment (RolaH1t) [inline docs/tractus-x-release.md:110]: add URL

File: docs/planning.md
Comment (Siegfriedk) [reply docs/planning.md:1]: Not sure if i get it? I assumed its english rule to have capital letters in a title. If you have specific feedback, pls provide it as i'm not a english grammer expert at all and need to lookup those rules adhoc

File: README.md
Comment (Siegfriedk) [reply README.md:13]: I don't think thats correct. Tractus-X is a name and we are referenzing a document with the same name. I fixed the link

File: docs/product_release.md
Comment (Siegfriedk) [reply docs/product_release.md:1]: You mean Semantic Versioning? or something else?

File: docs/product_release.md
Comment (Siegfriedk) [reply docs/product_release.md:52]: added url

File: docs/tractus-x-release.md
Comment (Siegfriedk) [reply docs/tractus-x-release.md:3]: As mentioned in the other comment, i would write Tractus-X upper case.

Also pls be more specific what issues you see. Its weird to read and diff the sentence to find this out

Comment (Siegfriedk) [reply docs/tractus-x-release.md:3]: Btw. we are repeating ourselves here. Its the same sentence as in README.md

File: docs/tractus-x-release.md
Comment (Siegfriedk) [reply docs/tractus-x-release.md:40]: I added the text, fixed the link template and shortend it to 231006_Whitepaper_Tractus-X.pdf

File: docs/tractus-x-release.md
Comment (Siegfriedk) [reply docs/tractus-x-release.md:42]: So we reference Tractus-X now as TX? Shouldn't we just keep Tracuts-X? Its short enough

Comment (Siegfriedk) [reply docs/tractus-x-release.md:42]: Text adjusted to suggestion

File: docs/tractus-x-release.md
Comment (Siegfriedk) [reply docs/tractus-x-release.md:44]: product_release.md doesn't contain any real description of a product development team. Why would we link to this?

File: docs/tractus-x-release.md
Comment (Siegfriedk) [reply docs/tractus-x-release.md:110]: done

File: README.md
Comment (RolaH1t) [reply README.md]: pls update to:
Tractus-X operates on a quarterly release cycle. Each quarter, a new [Tractus-X release] (tractus-x-release.md) following [CalVer](https://calver.org/) is released.
=> mind the necessary URL link in normal brackets

File: docs/planning.md
Comment (RolaH1t) [reply docs/planning.md:1]: thanks for checking / resolved

File: docs/product_release.md
Comment (RolaH1t) [reply docs/product_release.md]: thanks for checking / resolved

File: docs/tractus-x-release.md
Comment (RolaH1t) [reply docs/tractus-x-release.md]: no duplicates found. pls keep sentence.
pls update to:
Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release following [CalVer] (URL) is released. Each Tractus-X release contains multiple [products] (product_release.md).
hint: URL links shown in normal brackets

File: docs/tractus-x-release.md
Comment (RolaH1t) [reply docs/tractus-x-release.md:38]: still trying to obtain URL

File: docs/tractus-x-release.md
Comment (RolaH1t) [reply docs/tractus-x-release.md]: compromise proposal:
No matter if carry-over versions of a previous Tractus-X release or evolutions of the current iteration. User Journey tests (E2E) are automatically performed - customized for the respective bundle of products and within given boundary conditions.

File: docs/tractus-x-release.md
Comment (RolaH1t) [reply docs/tractus-x-release.md:44]: I still vote for my original proposal with relevant argumentation.
Leave the final call with you.

File: README.md
Comment (Siegfriedk) [reply README.md]: done

File: docs/tractus-x-release.md
Comment (Siegfriedk) [reply docs/tractus-x-release.md:38]: @RolaH1t url still missing; i keep it removed. We can do a patch if you get the url

File: docs/tractus-x-release.md
Comment (Siegfriedk) [reply docs/tractus-x-release.md:44]: added the link

**Comment by stephanbcbauer** [comment]
Like this PR and the description. Since the lifecycle management changed a little (Maybe not for Tractus-X, but I would like to check) and also the standards are not provided on the “expected” URL any more, I would like to review this PR ... but ... not this week. It would be planned for next week.

**Comment by stephanbcbauer** [comment]
@Grand-Thibault maybe we can review this together because some parts have to match the working model (at least the open-source part :))

File: README.md
Comment (lgblaumeiser) [inline README.md:17]: As I am working in the release management group in the association, the impact of the breaking change release thing is currently detailed. The clear requirement is, that the dataspace runs smoothly and companies can update in a defined grace period. This is defining "breaking changes" on a conceptual level. It has to be detailed, what this means for concrete developments. It is not possible to have breaking changes on the dataspace level, i.e., in the interactions companies have with other companies and especially with the operating company. So this statement here will break this requirement. Please wait until clarification is agreed upon.

**Review by lgblaumeiser — CHANGES_REQUESTED** [review]

File: README.md
Comment (tomaszbarwicki) [inline README.md:19]: Perhaps we can link to general tractus-x release changlog: https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md ?

File: docs/product_release.md
Comment (tomaszbarwicki) [inline docs/product_release.md:1]: I'd suggest to add a title (top level heading) to align to other docs
```
# Product Release
## Intro
```

File: docs/tractus-x-release.md
Comment (tomaszbarwicki) [inline docs/tractus-x-release.md:1]: Also adding doc title:

```
# Tractus-X Release
```

**Review by tomaszbarwicki — DISMISSED** [review]
Looks good to me in general, just few minor suggestions.

File: README.md
Comment (stephanbcbauer) [reply README.md:17]: fine for me... @lgblaumeiser lets sync after the PR is merged. Would be happy to contribute here

File: docs/product_release.md
Comment (Sicaine) [reply docs/product_release.md:5]: done

File: docs/tractus-x-release.md
Comment (Sicaine) [reply docs/tractus-x-release.md:3]: done

File: docs/tractus-x-release.md
Comment (Sicaine) [reply docs/tractus-x-release.md]: done

File: README.md
Comment (lgblaumeiser) [reply README.md:17]: Is ok, as the impact of the sentence is still quite far in the future.

Comment (Siegfriedk) [reply README.md:17]: @lgblaumeiser i changed sooo much, pls let us merge it now and finetune it next week with @stephanbcbauer

**Review by tomaszbarwicki — APPROVED** [review]

**Pull request merged**
